commercial/src/dig_components/companies_list.js

```diff
@@ -42,19 +42,22 @@ function CompaniesList(props) {
 				{companiesList.filter((company) => {
 					if(singleCompany == '') {
 						return company;
-					}else if(company.name.toLowerCase().includes(singleCompany.toLowerCase())) {
+					}else if(company.company_name.toLowerCase().includes(singleCompany.toLowerCase())) {
```

Comment:
I would recommend changing `singleCompany` to `companyNameSearchInput` or something like that